classes/py1.html

```diff
@@ -33,11 +33,14 @@ <h1>how to get started in<br />
     <div class="py1-divider">
     </div>
     <div class="py1-curriculum">
+        <!-- fundamentals lesson cards-->
+        <h2 class = "title">fundamentals</h2>
         <div class='py1-lessons-wrapper'>
             {% for lesson in site.py1 %}
+            {% if lesson.group == "fundamentals" %}
             <div class='py1-lesson'>
                 <div class='py1-lesson-title-bar'>
-                    <h2><a class='py1-link' href="{{ lesson.url }}">{{ lesson.title }}</a></h2>
+                    <h2><a class='py1-lesson-link' href="{{ lesson.url }}">{{ lesson.title }}</a></h2>
```

Comment:
This should probably be an `<h3>`, since it is under the `<h2>` category "fundamentals". Similar comment on the flow ones!